### PR TITLE
Add catalog command shell and AttachCommand

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

**Comment by ZacBlanco** [comment]
jenkins, test this please

File: experimental/table/shell/src/main/java/alluxio/cli/catalog/command/AttachDatabaseCommand.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  }
+
+  @Override
+  public String getDescription() {
+    return "Attaches a database to the alluxio catalog service from an under store DB";
+  }
+
+  @Override
+  public String getUsage() {
+    return "attachdb <dbName> <dbType> [-o|--option <option>]";
```
Comment (gpang) [inline experimental/table/shell/src/main/java/alluxio/cli/catalog/command/AttachDatabaseCommand.java]: is `<option>` more like `<key=value>`?

Comment (ZacBlanco) [reply experimental/table/shell/src/main/java/alluxio/cli/catalog/command/AttachDatabaseCommand.java]: I'll update the usage

**Comment by gpang** [comment]
Jenkins, test this please

**Comment by gpang** [comment]
alluxio-bot, merge this please

**Pull request merged**